<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Calculator v1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="VVOD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>310</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="QAC">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QCH_ZNAK">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QMOD">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QDELENIE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: orange; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QMULTIPLY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: orange; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QMINUS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: orange; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QPLUS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: orange; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QRAVNO">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: orange; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QCREDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: green; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>CR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QTOCHKA">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qint0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QDEBIT">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: green; }
QPushButton:hover {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(0, 0, 0, 0.1), stop: 1 #dadbde);}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(0, 0, 0, 0.3), stop: 1 #dadbde);
}
</string>
    </property>
    <property name="text">
     <string>DB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GRAPHIC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: ; }</string>
    </property>
    <property name="text">
     <string>GR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QKOREN">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QSKOBKA_R">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QSINUS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QCOSINUS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QTANGENS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QARKCOS">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QARKSIN">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QARKTAN">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QLOG10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QLOG">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QSKOBKA_L">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QIKS">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QSTEPEN">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
